docs/update-locking.md

```diff
@@ -2,7 +2,7 @@
 
 Locking updates means that the Supervisor will not be able to kill your application. This is meant to be used at critical sections of your code where you don't want to be interrupted, or to control that updates are only installed at certain times.
 
-In order to do this, users can create a file called `resin-updates.lock` that will prevent the Supervisor from killing and restarting the app. As any other lockfile, the Supervisor itself will create such file before killing the app, so you should only create it in exclusive mode. This means: only create the lockfile if it doesn't already exist. Several tools exist to simplify this process, for example [npm/lockfile](https://github.com/npm/lockfile).
+In order to do this, users take the lock on a file called `resin-updates.lock` that will prevent the Supervisor from killing and restarting the app. As any other lockfile, the Supervisor itself will create such file before killing the app, so you should only create it in exclusive mode. This means: only create the lockfile if it doesn't already exist. Several tools exist to simplify this process, for example [npm/lockfile](https://github.com/npm/lockfile).
```

Comment:
`users take the lock on a file` is very awkward wording and doesn't make much sense to me. Can this be improved?

Reply:
Is `Creating a lock` more semantic?

Reply:
`Creating a lock` suggests that creating the file is sufficient, I think what @imrehg wanted to make clearer is that the creation needs to happen with the O_EXCL flag to take the lock rather than just create the file in any way

Reply:
So maybe what we could do is clarify thet `take the lock` means `create the lockfile with O_EXCL and O_CREAT flags set`

Reply:
@pcarranzav I think that would definitely be an improvement. I think it's important that we don't make *too many* assumptions about the user's knowledge level here, any additional information/pointers we can provide is good. We certainly need to find a balance between verbosity and readability, however, the expression `take the lock` doesn't seem to be widely used or commonly known.